source/_docs/mqtt/broker.markdown

```diff
@@ -9,7 +9,7 @@ The MQTT integration needs you to run an MQTT broker for Home Assistant to conne
 ### Run your own
 
 This is the most private option, but it requires a little bit of work to setup. There are multiple free and open-source brokers to pick from: e.g., [Mosquitto](http://mosquitto.org/), [EMQ](https://github.com/emqx/emqx) or [Mosca](http://www.mosca.io/).
-For Hass.io users, the recommended setup method is to use the [Mosquitto MQTT broker addon](/addons/mosquitto).
+For Home Assistant users, the recommended setup method is to use the [Mosquitto MQTT broker addon](/addons/mosquitto), this guide describes the steps for Home Assistant Core users.
```

Comment:
We should move this to the first sentence. Since the "requires a little bit of work" does not apply.

Reply:
Simplified the guide overall.